src/model/webhook.rs

```diff
@@ -132,9 +180,14 @@ impl Webhook {
     ///
     /// # Errors
     ///
-    /// May return an [`Error::Http`] if the content is illformed, or if the token is invalid.
-    /// May also return an [`Error::Json`] if there is an error in deserializing Discord's response.
+    /// Returns an [`Error::Model`] if the [`token`] is `None`.
+    ///
+    /// May also return an [`Error::Http`] if the content is illformed, or if the token is invalid.
```

Comment:
```suggestion
    /// May also return an [`Error::Http`] if the content is malformed, or if the token is invalid.
```
It's more common to say *malformed*.